server/tests/rooms/game/index.test.ts

```diff
@@ -199,4 +201,18 @@ describe('inverting pending inventory', () => {
     expect(g.players['Curator'].pendingInvestments.culture).toBe(-3);
     expect(g.players['Curator'].inventory.culture).toBe(3);
   });
-});
\ No newline at end of file
+});
+
+describe('out of commission event', () => {
+  const g = new GameState(mockGameStateInitOpts(x => x, () => 10));
+  const outOfCommissionEntreprenuer = getAllMarsEvents()[7][0];
```

Comment:
grab the `outOfCommissionEntrepreneur` directly instead of relying on its position in array

Reply:
this code is already fixed in another pull request how just removing the changes to this file?

Reply:
I really don't know why it's in here at all.
I'll make that change. Thanks!